source/reference/api.txt

```diff
@@ -1,10 +1,30 @@
 .. _c2c-api:
 
+=======================
+mongosync API Endpoints
+=======================
+
 .. default-domain:: mongodb
 
-===========================
-``mongosync`` API Endpoints
-===========================
+:ref:`c2c-api-start`
+   Starts the synchronization between a source and destination cluster.
+
+:ref:`c2c-api-progress`
+   Returns either an updated status of the synchronization process or an error.
+
+:ref:`c2c-api-pause`
+   Pauses the current synchronization operation. 
+
+:ref:`c2c-api-resume`
+   Resumes a paused synchronization session based on data stored on the
+   destination cluster.
+
+:ref:`c2c-api-commit`
+   Finalizes the sync between the source cluster and the destination
+   cluster and stops continuous sync between clusters.
+
+:ref:`c2c-api-reverse`
+   Reverses the direction of a committed sync operation.
```

Comment:
Instead of saying "process", "operation" and "session", could we use only one of these to describe the sync? I think that "session" is most logically representative, but I'm okay with either of the other two.

Reply:
Done! I went with "session".